v03_pipeline/lib/reference_data/clinvar.py

```diff
@@ -128,18 +126,23 @@ def download_and_import_latest_clinvar_vcf(
 
     if genome_version not in ['37', '38']:
         raise ValueError('Invalid genome_version: ' + str(genome_version))
-    mt_contig_recoding = {'MT': 'chrM'} if genome_version == '38' else None
+
+    if genome_version == '38':
```

Comment:
I made a [ticket](https://github.com/broadinstitute/seqr-loading-pipelines/issues/606) to clean this up later, but thought it was out of scope for now